packages/plugin-core/src/components/lookup/LookupViewModel.ts

```diff
@@ -0,0 +1,30 @@
+import { TwoWayBinding } from "../../types/TwoWayBinding";
+import { VaultSelectionMode } from "./types";
+
+export enum NameModifierMode {
```

Comment:
Just curious. Are we moving away from calling these "note type modifiers" since it clashes with note traits?

Reply:
No good reason - I realize now that we've duplicated some of these enums in 3 places - I just did a pass at consolidating to the ones defined in common-all.